vsc-ci.ini

```diff
@@ -2,3 +2,4 @@
 pip3_install_tox=1
```

Comment:
i think none of the py3 things should be here anymore? not sure pip3 is similar, but py3 is now default
(it's possible that clusterbuildrpm server still looks for it, best to check that too)